packages/qwik/src/core/v2/ssr/ssr-render-jsx.ts

```diff
@@ -491,14 +490,17 @@ function getSlotName(host: ISsrNode, jsx: JSXNode, ssr: SSRContainer): string {
       return trackSignal(() => constValue.value, host as fixMeAny, EffectProperty.COMPONENT, ssr);
     }
   }
-  return (jsx.props.name as string) || QDefaultSlot;
+  return directGetPropsProxyProp(jsx.varProps, jsx.constProps, 'name') || QDefaultSlot;
 }
 
 function appendQwikInspectorAttribute(jsx: JSXNode) {
   if (isDev && qInspector && jsx.dev && jsx.type !== 'head') {
     const sanitizedFileName = jsx.dev.fileName?.replace(/\\/g, '/');
     const qwikInspectorAttr = 'data-qwik-inspector';
-    if (sanitizedFileName && !(qwikInspectorAttr in jsx.props)) {
```

Comment:
I would expect it's faster to do
```ts
if (sanitizedFileName && !(qwikInspectorAttr in jsx.varProps) && !(qwikInspectorAttr in jsx.constProps)) {
```

but actually we never put it in varProps so this could be
```ts
    if (
      sanitizedFileName &&
      !jsx.constProps || !(qwikInspectorAttr in jsx.constProps))
    ) {
      (jsx.constProps ||= {})[qwikInspectorAttr] =
        `${sanitizedFileName}:${jsx.dev.lineNumber}:${jsx.dev.columnNumber}`;
    }
```